scripts/zones/Balgas_Dais/mobs/King_of_Batons.lua

```diff
@@ -0,0 +1,36 @@
+-----------------------------------
+-- Area: Balga's Dais
+-- NM: King of Batons
+-- KSNM: Royale Ramble
+-----------------------------------
+mixins = {require("scripts/mixins/job_special")}
+require("scripts/globals/status")
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:SetMod(xi.mod.DOUBLE_ATTACK, 50)
```

Comment:
CI checks are failing for all of the SetMod which should be setMod

Reply:
the dangers of copy past